uefi_app/SbsaAvsMain.c

```diff
@@ -256,7 +256,7 @@ HelpMsg (
   VOID
   )
 {
-  Print (L"\nUsage: Sbsa.efi [-v <nn>] | [-l <n>] | [-f <filename>] | "
+  Print (L"\nUsage: Sbsa.efi [-v <n>] | [-l <n>] | [-f <filename>] | [-skip <n>] | [-nist] | [-p <n>] | [-t <n>] | [-m <n>]\n"
```

Comment:
Can skip, -t and -m option be used together?
If not, we have to indicate on the commandline the they are exclusive.

Reply:
Yes, these should be usable together. We often use -skip and -m in the same invocation.